### PR TITLE
Implement new Toolbar in person overview (EXPOSUREAPP-13260)

### PR DESCRIPTION
**Pull request by mtwalli**
- Adjust toolbar to allow export all certs 

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13260

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml [hunk 1]
```diff
@@ -348,4 +348,9 @@
     <string name="certificate_migration_dialog_title">Zuordnung von Zertifikaten</string>
     <!-- XTXT: Certificate Migratiomn Dialog Message -->
     <string name="certificate_migration_dialog_message">Die Gruppierung von Zertifikaten wurde verbessert. Zertifikate mit geringen Namensunterschieden werden nicht mehr verschiedenen Personen zugeordnet.</string>
+
+    <!-- XTXT: Certificate export all tooltip title -->
+    <string name="export_all_tooltip_title">Zertifikate Exportieren</string>
```
Comment (KathrinLuecke) [inline Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml:353]:
```suggestion
    <string name="export_all_tooltip_title">Zertifikate exportieren</string>
```

**Comment by Ein-Tim** [comment]
May we get a screenshot of this change, please?

**Review by SamuraiKek — APPROVED** [review]
LGTM.

**Comment by mtwalli** [comment]
> May we get a screenshot of this change, please?

Screenshot 2022-06-02 at 14 13 50

**Pull request merged**